guillotina/contrib/swagger/services.py

```diff
@@ -91,11 +91,12 @@ def get_endpoints(self, iface_conf, base_path, api_def, tags=None):
         for method in iface_conf.keys():
             if method == "endpoints":
                 for name in iface_conf["endpoints"]:
+                    for http_method in iface_conf["endpoints"][name].keys():
```

Comment:
How does this change it?

It's unclear to me what the data structure is here so we need to do:

```
tags = iface_conf["endpoints"][name][http_method][key]
```

Reply:
the tags were never actually fetched from the passed in data so we always ended up with the name of the endpoint as the tag. in the calling code https://github.com/plone/guillotina/blob/219230961065b5b56a960a819ceacd050fc5ac51/guillotina/contrib/swagger/services.py#L166, tags are not passed in, so always defaults to `None`. this change iterates over the service definition for each endpoint (`name`) and then for each http method looks for the tags.